src/System.Net.NameResolution/src/System/Net/NameResolutionPal.Unix.cs

```diff
@@ -139,24 +139,6 @@ private static unsafe IPHostEntry CreateIPHostEntry(Interop.Sys.HostEntry hostEn
             };
         }
 
-        public static unsafe IPHostEntry GetHostByName(string hostName)
-        {
-            if (hostName == "")
-            {
-                // To match documented behavior on Windows, if an empty string is passed in, use the local host's name.
-                hostName = Dns.GetHostName();
-            }
-
-            Interop.Sys.HostEntry entry;
-            int err = Interop.Sys.GetHostByName(hostName, &entry);
-            if (err != 0)
-            {
-                throw SocketExceptionFactory.CreateSocketException(GetSocketErrorForErrno(err), err);
-            }
-
-            return CreateIPHostEntry(entry);
-        }
-
         public static unsafe IPHostEntry GetHostByAddr(IPAddress addr)
```

Comment:
You removed the only call to this, right?  Seems like this enables a bunch more code to be removed, all the way down to the native layer.

Reply:
Ok i'll work on It!

Reply:
@geoffkizer  i did some check and: 
* Dns.GetHostName() used by TryGetAddrInfo();
* Interop.Sys.GetHostByName(hostName, &entry) removed(pal_networking.h etc...)
* CreateIPHostEntry used by TryGetAddrInfo() and others
the other methods of this class are used by System.Net.Dns, what i'm missing?

Reply:
I mean specifically NameResolutionPal.GetHostByAddr.  I don't think this is used anymore, right?

Reply:
sorry i don't know why but my eyes missed that line and check only method...maybe i need a eye doctor, anyway you're right, i'll work on that.